manifest.json

```diff
@@ -1,26 +1,31 @@
 {
   "name": "DarkCloud",
   "description": "SoundCloud Dark Theme",
-  "version": "2023.11.07",
-  "permissions": [],
-  "icons": {"128": "darklogo.png" },
+  "version": "2024.07.30",
+  "permissions": ["scripting"],
+  "manifest_version": 3,
+  "icons": {
+    "128": "darklogo.png"
+  },
   "background": {
-    "scripts": ["background.js"]
+    "service_worker": "background.js"
```

Comment:
I don't think this is required given what this plugin is doing.

`{
  "name": "DarkCloud",
  "description": "SoundCloud Dark Theme",
  "version": "2024.07.30",
  "permissions": ["scripting"],
  "manifest_version": 3,
  "icons": {
    "128": "darklogo.png"
  },
  "action": {
    "default_title": "DarkCloud",
    "default_icon": "darklogo.png"
  },
  "web_accessible_resources": [
    {
      "resources": ["css_dark.css"],
      "matches": ["https://soundcloud.com/*"]
    }
  ],
  "content_scripts": [
    {
      "matches": ["https://soundcloud.com/*"],
      "js": ["background.js"],
      "css": ["css_dark.css"],
      "run_at": "document_start"
    }
  ]
}`